src/authorizations/utils/permissions.ts

```diff
@@ -329,14 +329,20 @@ export const generateDescription = apiPermissions => {
     }
   } else {
     apiPermissions.forEach(perm => {
-      if (perm.resource.name) {
+      if (perm.resource.type === 'orgs') {
         generatedDescription += `${capitalize(perm.action)} ${
           perm.resource.type
-        } ${perm.resource.name} `
+        }  `
       } else {
-        generatedDescription += `${capitalize(perm.action)} ${
-          perm.resource.type
-        } `
+        if (perm.resource.name) {
+          generatedDescription += `${capitalize(perm.action)} ${
+            perm.resource.type
+          } ${perm.resource.name} `
+        } else {
+          generatedDescription += `${capitalize(perm.action)} ${
+            perm.resource.type
+          } `
```

Comment:
I originally left a comment on the this about structure, but after looking over it more clearly (like I should have done _before_ leaving the comment), I realized the comment was wrong because it didn't take into account the concatenation operator. My bad!